lib/reporting/proofing_rate_report.rb

```diff
@@ -8,20 +8,22 @@ module Reporting
   class ProofingRateReport
     DATE_INTERVALS = [30, 60, 90].freeze
 
-    attr_reader :end_date, :wait_duration
+    attr_reader :end_date, :wait_duration, :by_month
 
     def initialize(
       end_date:,
       verbose: false,
       progress: false,
       wait_duration: CloudwatchClient::DEFAULT_WAIT_DURATION,
-      parallel: false
+      parallel: false,
+      by_month: false
```

Comment:
`by_month` causes us to use the past 3 calendar months versus trailing `DATE_INTERVALS` (30/60/90 days). This is what I want in MonthlyIdvReport.

Reply:
Do we need to keep the old DATE_INTERVALS method?

Reply:
Yes -- the existing report will still be used that way, and ours will use calendar months. `by_month` is the toggle, defaulting to false.

Reply:
Removed this code (aside from some lingering stuff to clean up).